devel/haskell-language-server/Portfile

```diff
@@ -0,0 +1,162 @@
+                    for the Language Server Protocol (LSP). A language \
+                    server talks to a client (typically an editor), \
+                    which can ask the server to perform various \
+                    operations, such as reporting errors or providing \
+                    code completions. The advantage of this system is \
+                    that clients and servers can interoperate more \
+                    easily so long as they all speak the LSP \
+                    protocol. In the case of HLS, that means that it \
+                    can be used with many different editors, since \
+                    editor support for the LSP protocol is now \
+                    widespread.
+
+master_sites        https://hackage.haskell.org/package/${name}-${version}
+
+checksums           rmd160  66c0d520c28d2a8e4a1ca0e3665f2aa73ed75582 \
+                    sha256  3655b8d741616f8e32d0181274f80a48d9b8dca47d3fd22675cadfa01ca80e48 \
+                    size    80146
+
+set name_underscore [string map {- _} ${name}]
+set paths_name_underscore   src/Paths_${name_underscore}.hs
+
+# relative paths to ${prefix}
+set hls_libdir      lib/${distname}
+set hls_docdir      share/doc/${distname}/html
+
+
+post-extract {      
+    # Fix for cabal data-files hardcoded path in binary
+    # See:          
+    # https://github.com/commercialhaskell/stack/issues/848
+    # https://github.com/commercialhaskell/stack/issues/4857
+    # https://github.com/haskell/cabal/issues/462
+    # https://github.com/haskell/cabal/issues/3586
+    xinstall -m 0644 -W ${worksrcpath} \
+                    ${filespath}/Paths_NAME.hs ${paths_name_underscore}
+
+    reinplace "s|@PREFIX@|${prefix}|g" \
+                    ${worksrcpath}/${paths_name_underscore}
+    reinplace "s|@NAME@|${name_underscore}|g" \
+                    ${worksrcpath}/${paths_name_underscore}
+    reinplace -E "s|(Version\[\[:space:\]\]+)\\\[\[\[:digit:\]\]+\[\[:space:\]\]*(,\[\[:space:\]\]*\[\[:digit:\]\]+){1,4}\[\[:space:\]\]*\\\]|\\1\[[join [split ${version} .] ,]\]|" \
+                    ${worksrcpath}/${paths_name_underscore}
+}                   
+
+depends_build-append \
+                    port:file
+
+# see ${name}.cabal for build specifics
+# install depends on ghc version libraries, not simply `path:bin/ghc:ghc`
+depends_lib-append  port:ghc
+
+build.target
+build.post_args-prepend \
+                    -fdynamic
+
+destroot.post_args-prepend \
+                    -fdynamic
+
+set hls_build_binary    {}
+set hls_build_doc       {}
+pre-destroot {
+    set hls_build_binary \
+        [glob -nocomplain ${workpath}/dist/build/*/ghc-*/${name}-${version}/x/${name}/build/${name}/${name}]
+    set hls_build_doc \
+        [glob -nocomplain ${workpath}/dist/build/*/ghc-*/${name}-${version}/doc/html/${name}]
+}
+
+destroot {
+    xinstall -d     ${destroot}${prefix}/${hls_libdir} \
+                    ${destroot}${prefix}/${hls_docdir}
```

Comment:
no need for aligning this, just continue after the `-d`

Reply:
Thanks, fixed.